op_crates/fetch/26_fetch.js

```diff
@@ -1297,82 +1416,74 @@
    * @returns {Promise<Response>}
    */
   async function fetch(input, init) {
```

Comment:
Oddly enough, Chrome says `fetch.length === 0`, even though it also has the length === 1 assertion.

WebIDL:
```
[NewObject] Promise<Response> fetch(RequestInfo input, optional RequestInit init = {});
```

Can someone check Firefox and see what theirs says? If they also say 0, then I need to review the spec and figure out how this is possible.

Reply:
FF:
```
> fetch.length
1
```

Chrome is wrong.

Reply:
Thanks! I'll go open an bug report with them.

Reply:
Ignore this, I was totally off, Chrome says `1`, the site (not GitHub) corrupted their own `fetch`. Annoying.